cli/pkg/gitutil/gitcmdwrapper.go

```diff
@@ -193,9 +194,9 @@ func isGitIgnored(repoRoot, subpath string) (bool, error) {
 				return false, nil
 			}
 			// any other exit code is an error
-			return false, fmt.Errorf("git check-ignore failed: %s", string(execErr.Stderr))
+			return false, fmt.Errorf("git check-ignore failed for path %q, subpath: %q : %s", repoRoot, subpath, string(execErr.Stderr))
 		}
-		return false, fmt.Errorf("git check-ignore failed: %w", err)
+		return false, fmt.Errorf("git check-ignore failed for path %q, subpath: %q : %w", repoRoot, subpath, err)
```

Comment:
nit: try the format `path %q, subpath %q: %w` – colons should be used for different levels of context, not for key-value pairs; and there should not be a space before a colon.